examples/tree_2d_dgsem/elixir_eulermulti_shock_bubble_7components.jl

```diff
@@ -1,70 +0,0 @@
-using OrdinaryDiffEq
```

Comment:
Why did you delete this file?

Reply:
It's a testcase I have developed for testing purpose, it's not necessary anymore.